packages/ui/cypress/tests/watched-acccounts/watched-accounts.cy.ts

```diff
@@ -0,0 +1,62 @@
+import { addresses } from '../../fixtures/accounts'
+import { localHost } from '../../fixtures/landingData'
+import { landingPage } from '../../support/page-objects/landingPage'
+import { settingsPage } from '../../support/page-objects/settingsPage'
+
+describe('Watched Accounts', () => {
+  it('can add an account to the watch list', () => {
+    cy.visit(localHost)
+    landingPage.watchAccountButton().click()
+    settingsPage.accountAddressInput().type(`${addresses.Alice}{enter}`)
+    // autocomplete on the input seems to affect reliability so we wait a bit
+    cy.wait(1000)
+    settingsPage.accountNameInput().type('{selectall}{del}Alice')
+    settingsPage.addButton().click()
+    settingsPage.accountContainer().within(() => {
+      settingsPage.accountIcon().should('be.visible')
+      settingsPage.accountAddressLabel().should('be.visible')
+      settingsPage.accountNameLabel().should('be.visible')
+      settingsPage.accountDeleteButton().should('be.visible')
+    })
+  })
+
+  it('can remove an account from the watch list', () => {
+    cy.window().then((win) => {
+      win.localStorage.setItem('multix.watchedAccount', `["${addresses.AliceSS58}"]`)
+    })
+    cy.visit(localHost)
+    landingPage.watchAccountButton().click()
+    settingsPage.accountContainer().within(() => {
+      settingsPage.accountDeleteButton().click()
+      settingsPage.accountIcon().should('not.exist')
+      settingsPage.accountAddressLabel().should('not.exist')
+    })
+    settingsPage.accountContainer().should('have.length', 0)
+  })
+
+  it('can see error when attemping to add same address more than once', () => {
+    // stub existing watched account
+    cy.window().then((win) => {
+      win.localStorage.setItem('multix.watchedAccount', `["${addresses.AliceSS58}"]`)
```

Comment:
I think this is dangerous to do as it is messing with localstorage, that shouldn't be messed with. Localstorage isn't something that should be tested. We should manually add an account, and followup by checking we can't add it twice.

I think it's extremely unlikely that someone pastes the public key (I think the name here is misleading BTW, ss58 is something different). Maybe we should have Alice with some other ss58 encoding?

Reply:
Updated to do everything within the UI now